Tests/SmithyIdentityTests/StaticAWSCredentialIdentityResolverTests.swift

```diff
@@ -9,14 +9,17 @@ import XCTest
 import struct SmithyIdentity.StaticAWSCredentialIdentityResolver
 
 class StaticAWSCredentialIdentityResolverTests: XCTestCase {
+
     func testGetCredentials() async throws {
         let subject = try StaticAWSCredentialIdentityResolver(.init(
-            accessKey: "some_access_key",
-            secret: "some_secret"
+            accessKey: "static_access_key",
+            secret: "static_secret",
+            accountID: "static_account_id"
```

Comment:
Like in the mock above, values are marked with `static` to make clear where they originated.